src/main/java/com/streamr/client/options/ResendFromOption.java

```diff
@@ -0,0 +1,42 @@
+package com.streamr.client.options;
+
+import com.streamr.client.protocol.control_layer.ControlMessage;
+import com.streamr.client.protocol.control_layer.ResendFromRequest;
+import com.streamr.client.protocol.message_layer.MessageRef;
+
+public class ResendFromOption extends ResendOption {
+    private MessageRef from;
+    private String publisherId;
+    private String msgChainId;
+
+    public ResendFromOption(MessageRef from, String publisherId, String msgChainId) {
+        this.from = from;
+        this.publisherId = publisherId;
+        this.msgChainId = msgChainId;
+    }
+
+    public ResendFromOption(long timestamp, long sequenceNumber, String publisherId, String msgChainId) {
```

Comment:
Would be nicer to use `Date`s for timestamps in developer-facing interfaces, even if the protocol uses `long`s internally